api_client/python/timesketch_api_client/user.py

```diff
@@ -43,6 +45,38 @@ def _get_data(self):
 
         return self._object_data
 
+    def change_password(self, new_password=''):
```

Comment:
Do we really want CLI user interface in the API client? Isn't this the responsibility of the user frontend client (e.g. Colab magics, picatrix, the CLI client)?

Reply:
sure, let's remove the `cli_input` logic